R/convergence.R

```diff
@@ -316,6 +316,144 @@ quantile2 <- function(x, probs = c(0.05, 0.95), names = TRUE, ...) {
   out
 }
 
+#' Calculate R star convergence diagnostic
```

Comment:
Since rstar is such a different diagnostic than the others, I think it makes sense to move the code to a separate R file (called `rstar.R`).

Reply:
Done.